packages/express-did-auth/src/index.ts

```diff
@@ -40,6 +41,10 @@ export default function setupAppFactory (config: ExpressDidAuthConfig) {
     if (config.useCookies) {
       app.use(cookieParser())
       app.use(csrf({ cookie: true }))
```

Comment:
Lets just play it safe
```suggestion
      app.use(csrf({ cookie: true, httpOnly: true }))
```